python/plugins/processing/core/ProcessingConfig.py

```diff
@@ -83,6 +84,10 @@ def initialize():
             ProcessingConfig.tr('Use only selected features'), True))
         ProcessingConfig.addSetting(Setting(
             ProcessingConfig.tr('General'),
+            ProcessingConfig.IGNORE_NULL_GEOMS,
+            ProcessingConfig.tr('Ignore features with null geometries'), True))
```

Comment:
Just a note - I spent a long time working in GIS in law enforcement, and the datasets we used there typically contained ~ 25% null geometries. It's *really* important not to treat null geometries only as some form of corrupted data, but rather they need to be treated just as importantly as every other feature. 

For me +1 to @m-kuhn 's approach